etl/steps/export/explorers/agriculture/latest/crop_yields.config.yml

```diff
@@ -819,3 +980,603 @@ views:
     config:
       type: LineChart
       note: *attainable_yield_footnote
+  - dimensions:
+      crop: chickpeas
+      metric: actual_yield
+    indicators:
+      y:
+        - catalogPath: attainable_yields#chickpeas_yield
+          display:
+            name: Chickpea yield
+  - dimensions:
+      crop: kiwi
+      metric: actual_yield
+    indicators:
+      y:
+        - catalogPath: attainable_yields#kiwi_yield
+  - dimensions:
+      crop: linseed
+      metric: actual_yield
+    indicators:
+      y:
+        - catalogPath: attainable_yields#linseed_yield
+  - dimensions:
+      crop: leeks
+      metric: actual_yield
+    indicators:
+      y:
+        - catalogPath: attainable_yields#leeks_yield
+          display:
+            name: Leek yield
+  - dimensions:
+      crop: tangerines
+      metric: actual_yield
+    indicators:
+      y:
+        - catalogPath: attainable_yields#tangerines_yield
+          display:
+            name: Tangerine yield
+  - dimensions:
+      crop: avocados
+      metric: actual_yield
+    indicators:
+      y:
+        - catalogPath: attainable_yields#avocados_yield
+          display:
+            name: Avocado yield
+  - dimensions:
+      crop: papayas
+      metric: actual_yield
+    indicators:
+      y:
+        - catalogPath: attainable_yields#papayas_yield
+          display:
+            name: Papaya yield
+  - dimensions:
+      crop: hempseed
+      metric: actual_yield
+    indicators:
+      y:
+        - catalogPath: attainable_yields#hempseed_yield
+  - dimensions:
+      crop: cow_peas
+      metric: actual_yield
+    indicators:
+      y:
+        - catalogPath: attainable_yields#cow_peas_yield
+          display:
+            name: Cow pea yield
+  - dimensions:
+      crop: cinnamon_and_cinnamon_tree_flowers__raw
+      metric: actual_yield
+    indicators:
+      y:
+        - catalogPath: attainable_yields#cinnamon_and_cinnamon_tree_flowers__raw_yield
+          display:
+            name: Cinnamon yield
+  - dimensions:
+      crop: chillies_and_peppers
+      metric: actual_yield
+    indicators:
+      y:
+        - catalogPath: attainable_yields#chillies_and_peppers_yield
+          display:
+            name: Chillies and peppers yield
```

Comment:
Chilli and pepper yield

Reply:
I think I had some conversation about these ones in the past. In American English, it should be "Chili". And I think that, because we are talking about multiple types of crops, maybe it should be in plural, e.g. "Chilies and peppers yield"? (I'm not sure)